server/schema/article/article.query.js

```diff
@@ -165,6 +166,17 @@ module.exports = new GraphQLObjectType({
       },
       resolve: searchArticle,
     },
+    countTotalArticles: {
+      description: 'Counts the total number of articles',
+      type: GraphQLInt,
+      args: {
+        onlyPublished: {
+          description: 'Whether to only list published articles',
+          type: GraphQLBoolean,
+        },
+      },
```

Comment:
Think if this is a use-case

If I am a user who has access to query restricted articles also, but I want to see the count of non-restricted articles?

Reply:
This use case does not occur, at least at the moment since users don't pick which articles to see and don't have any filter options. If they are allowed to see an article, it is shown naturally, if not, they do not realize that some articles are not being shown.